services/blockchain-app-registry/shared/utils/chain.js

```diff
@@ -23,14 +23,11 @@ const MYSQL_ENDPOINT = config.endpoints.mysql;
 
 const applicationMetadataIndexSchema = require('../database/schema/application_metadata');
 
-const getApplicationMetadataIndex = () => getTableInstance(
-	applicationMetadataIndexSchema.tableName,
-	applicationMetadataIndexSchema,
-	MYSQL_ENDPOINT,
-);
+const getApplicationMetadataTable = () => getTableInstance(
+	applicationMetadataIndexSchema, MYSQL_ENDPOINT);
```

Comment:
```suggestion
const getApplicationMetadataTable = () => getTableInstance(applicationMetadataTableSchema, MYSQL_ENDPOINT);
```

Reply:
One liner is not possible here since eslint-max length is set to 100 characters.
Went through the entire codebase to validate if we have one liners wherever possible for getTableInstance function.